### PR TITLE
added new language support for Romanian

### PR DESCRIPTION
**Pull request by b-dorian**
Added the text "(optional)" to the Romanian version of the Import button, as non tech-savy users found the Import part confusing. They were not sure if the Import part was mandatory for the app to run effectively.

### CODE REVIEW

File: app/locales/ro/import.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+    "import_title":"Importa Locatii (optional)",
+    "import_step_1":"1. Daca aveti locatii anterioare salvate in contul Google, le puteti importa logandu-va in contul Google si downloadand Istoricul Locațiilor",
```
Comment (razvangeangu) [inline app/locales/ro/import.json:3]: logandu-va in contul Google => prin autentificare la contul Google
downloadand => descarcand

File: app/locales/ro/import.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+    "import_step_2":"2. Deschideti aceasta sectiune din nou dupa ce ati downloadat istoricul. Datele se vor importa automat."
```
Comment (razvangeangu) [inline app/locales/ro/import.json:4]: downloadat => descarcat

File: app/locales/ro/notification.json [hunk 3]
```diff
@@ -0,0 +1,8 @@
+{
+    "notification_main_text": "Verificati-va intersectarile in functie de zi",
+    "notification_title": "Analizati-va intersectarile",
+    "notification_data_not_available": "\n\nNu aveti trasee downloadate!\n\n",
```
Comment (razvangeangu) [inline app/locales/ro/notification.json:4]: downloadate => descarcate

Comment (b-dorian) [reply app/locales/ro/notification.json]: modified, thanks

**Comment by razvangeangu** [comment]
Everything looks good

**Comment by kenpugsley** [comment]
Thank you for this translation.  The UI has changed significantly and has now finalized for the v1.0 release, which has resulted in a great deal of changes to the text in the final app.  Because of this we found that it was best to remove the existing translations and start again.  Even though we are also closing this PR, we hope you understand and will help us again.

The text in the app is now complete, so **we're asking for your help to redo this translation on the text for the v1.0 release**.  Our translation lead @alpita-masurkar will be coordinating this effort.

Thank you from the COVID Safe Paths dev team.

**Pull request closed without merging**